docusaurus/docs/advanced-usage/_category_.json

```diff
@@ -1,6 +0,0 @@
-{
-    "position": 5,
-    "label": "Advanced Usage",
```

Comment:
Slightly concerned by losing this section as it may clutter other sections. But not a blocker for me personally.

Reply:
In the doc plan, I relocated the content under the "Advanced" category to the main section for development. My feeling was that "advanced" categories have the potential to be catchalls and not terribly useful... and "usage" seemed a bit odd because the topics pertained to development. I'm open to keeping this section if we think it would be useful, but perhaps we can find a better word than "usage".